src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/MSBuildProject.cs

```diff
@@ -260,7 +260,7 @@ public TargetFrameworkMoniker TFM
 
                     if (propCount != 1)
                     {
-                        _logger.LogCritical("SDK projects being upgraded must have exactly one TargetFramework property. Found {TargetFrameworkCount} TargetFramework properties.", propCount);
+                        _logger.LogError("SDK projects being upgraded must have exactly one TargetFramework property. Found {TargetFrameworkCount} TargetFramework properties.", propCount);
```

Comment:
Why change the threshold on these log messages?

Reply:
I thought error was more appropriate than critical now because there is a problem (the tool can't handle a particular problem), but it's no longer a fatal problem (because we can move on to other projects).